### ISSUE TITLE
Attaching to ... ?

### ISSUE THREAD

**Issue by AnasGhrab** [Parisson/TimeSide#84]
No change :

```
Successfully installed PyYAML-3.11 docker-compose-1.4.0 docker-py-1.3.1 dockerpty-0.3.4 docopt-0.6.2 requests-2.6.2 texttable-0.8.3 websocket-client-0.32.0
MacBook-Pro-de-Anas:TimeSide anas$ docker-compose up
Starting timeside_log_1...
Starting timeside_lib_1...
Starting timeside_db_1...
Starting timeside_rabbitmq_1...
Starting timeside_static_1...
Starting timeside_home_1...
Starting timeside_app_1...
Starting timeside_nginx_1...
Starting timeside_worker_1...
Attaching to 
```

**Comment by yomguy** [comment]
please cleanup containers:
```docker-compose kill; docker-compose rm```
and retry..

**Comment by AnasGhrab** [comment]
No change : 
```
MacBook-Pro-de-Anas:TimeSide anas$ docker-compose kill; docker-compose rm
Going to remove timeside_worker_1, timeside_nginx_1, timeside_app_1, timeside_home_1, timeside_static_1, timeside_db_1, timeside_lib_1, timeside_log_1, timeside_rabbitmq_1
Are you sure? [yN] y
Removing timeside_worker_1... done
Removing timeside_nginx_1... done
Removing timeside_app_1... done
Removing timeside_home_1... done
Removing timeside_static_1... done
Removing timeside_db_1... done
Removing timeside_lib_1... done
Removing timeside_log_1... done
Removing timeside_rabbitmq_1... done
MacBook-Pro-de-Anas:TimeSide anas$ docker-compose up
Creating timeside_log_1...
Creating timeside_lib_1...
Creating timeside_db_1...
Creating timeside_rabbitmq_1...
Creating timeside_static_1...
Creating timeside_home_1...
Creating timeside_app_1...
Creating timeside_nginx_1...
Creating timeside_worker_1...
Attaching to 
```

**Comment by yomguy** [comment]
Sad docker bug :(

**Comment by yomguy** [comment]
Restart docker ?

**Comment by yomguy** [comment]
Fixed with docker-compose 1.4.2
https://github.com/docker/compose/issues/2067#issuecomment-142370015

**Comment by AnasGhrab** [comment]
Tnx
